HopsanGenerator/src/generators/HopsanFMIGenerator.cpp

```diff
@@ -1310,6 +1310,31 @@ bool HopsanFMIGenerator::generateToFmu(QString savePath, ComponentSystem *pSyste
     fmuHopsanSourceCode.replace("<<<setdataptrs>>>", setDataPtrsString);
     fmuHopsanSourceCode.replace("<<<timestep>>>", QString::number(pSystem->getDesiredTimeStep()));
 
+    QList<ParameterSpecification> parSpecs;
+    getParameters(parSpecs, pSystem);
+    QString addParametersToMap;
+    for(const auto &parSpec : parSpecs) {
+        QString dataType;
+        if(parSpec.type == "Real") {
+            addParametersToMap.append(QString("realParametersMap.insert(std::pair<int,hopsan::HString>(%1,\"%2\"));").arg(vr).arg(parSpec.name.toStdString().c_str()));
```

Comment:
It would be nice with a  \n after these map insertions, to make the generated code more readbale, now all all ow them end up on the same very long line

Reply:
Fixed this, and also some missing indentations.